webapp/src/service/AnonymousIdService.ts

```diff
@@ -12,7 +12,6 @@ export const AnonymousIdService = {
   reset() {
     return localStorage.setItem(
       ANONYMOUS_ID_LOCAL_STORAGE_KEY,
-      // @ts-ignore
       crypto.randomUUID()
```

Comment:
Logging has disappeared from #1625 tests but, in my memories, there was indeed an issue with randomUUID.

Reply:
yeah that seems related to why I upgraded typescript, so I assumed I could remove this ts ignore